DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -218,7 +218,12 @@
   <ioread sourceClass = "pat::Jet" version="[1-11]" targetClass="pat::Jet" source="int partonFlavour_" target="jetFlavourInfo_">
     <![CDATA[jetFlavourInfo_ = reco::JetFlavourInfo(0,onfile.partonFlavour_);]]>
   </ioread>
-  <class name="pat::MET"  ClassVersion="14">
+  <class name="pat::MET"  ClassVersion="19">
+    <version ClassVersion="19" checksum="428901429"/>
+    <field name="corMap_" transient="true"/>
+   <version ClassVersion="17" checksum="428901429"/>
```

Comment:
there is no need to put intermediate class versions here.
just put only one new version =15 with 428901429.
I don't understand why there is a version=17 and version=19 with the same checksum: this seems an unnecessary new version value.